.changeset/modern-dodos-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/rss': patch
```

Comment:
This would probably need to be a major version change as it would break someone’s site if they currently have posts with `drafts: true` and expect those to be included in output.

```suggestion
'@astrojs/rss': major
```

If we don’t think this merits a v2 release of the RSS package, one alternative would be to make _including_ the drafts the default for backwards compatibility and introduce filtering behind an option. _Not_ including drafts does seem like the sensible default though, so perhaps a v2 is warranted.